pinot-core/src/test/java/org/apache/pinot/core/query/executor/QueryExecutorExceptionsTest.java

```diff
@@ -155,8 +156,8 @@ public void setUp()
     resourceUrl = getClass().getClassLoader().getResource(QUERY_EXECUTOR_CONFIG_PATH);
     assertNotNull(resourceUrl);
     PropertiesConfiguration queryExecutorConfig = new PropertiesConfiguration();
-    queryExecutorConfig.setDelimiterParsingDisabled(false);
-    queryExecutorConfig.load(new File(resourceUrl.getFile()));
+    FileHandler fileHandler = new FileHandler(queryExecutorConfig);
```

Comment:
Use `CommonsConfigurationUtils.loadFromFile(resourceUrl.getFile())`, same for other places where config needs to be loaded